src/components/Footer/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import './Footer.css';
+import LOGO from '../../assets/img/craneLogo.png';
```

Comment:
let's use the dir name `images` - not the short form

Reply:
Let's use `SVG` for logos wherever possible

Reply:
@ckwagaba this has been worked on and fixed.